### DISCUSSION TITLE
how to set priorityClass for spark pod in k8s?

### DISCUSSION THREAD

**Discussion by user-GitHub-user** [apache/kyuubi#6433] [Q&A]
hello, i want to set priorityClass for kyuubi-spark, How should I configure it？
if i config driver.podTemplateFile and executor.podTemplateFile in kyuubi-default.conf, Will core and memory be replaced automatically？

**Comment by pan3793** [comment]
What does "kyuubi-spark" mean? Which Pod or component you are referring to?

**Reply by user-GitHub-user** [reply]
It means spark pod started from kyuubi, I found the configuration in spark documentation, thank you for answer
